estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,100 @@
+            <form string="Estate Properties">
+                <sheet>
+                    <div class="title">
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                        <field name="tag_ids" widget="many2many_tags"/>
+                    </div>
+                    <div>
+                        <group>
+                            <group>
+                                <field name="property_type_id"/>
+                                <field name="postcode"/>
+                                <field name="date_availability"/>
+                            </group>
+                            <group>
+                                <field name="expected_price"/>
+                                <field name="selling_price"/>
+                                <field name="best_price"/>
+                            </group>
+                        </group>
+                    </div>
+                    <notebook>
+                        <page string="Description">
+                            <group>
+                                <field name="description"/>
+                                <field name="bedrooms"/>
+                                <field name="living_area"/>
+                                <field name="facades"/>
+                                <field name="garage"/>
+                                <field name="garden"/>
+                                <field name="garden_area"/>
+                                <field name="garden_orientation"/>
+                                <field name="total_area"/>
+                            </group>
+                        </page>
+                        <page string="Offers">
+                            <field name="offer_ids"/>
+                        </page>
+                        <page string="Other Info">
+                            <group>
+                                <field name="salesperson_id"/>
+                                <field name="buyer_id"/>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="estate_properties_search_view" model="ir.ui.view">
+        <field name="name">estate.properties.search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Search Properties">
+                <field name="name"/>
+                <field name="property_type_id"></field>
+                <field name="postcode"/>
+                <field name="expected_price"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="facades"/>
+                <separator/>
+                <filter string="State" name="state" domain="[('state', '=', 'new'), ('state', '=', 'offer received')]"/>
```

Comment:
domain won't work here because it checks if the `state = new AND state = offer received`
2 possibilities : either you add an `OR` operator `['|', (state = new), (state = offer received)]`  (the syntax is a bit strange, it gets the operator first then the elements it has to compare. If no operator is specified, odoo sets an `AND` by default)
Second choice: you combine the 2 conditions in one: `domain="[('state', 'in', ['new', 'offer received'])]"`